src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -127,7 +128,7 @@ enum Gender
     GENDER_NONE                        =  2
 };
 
-// ChrRaces.dbc (6.0.2.18988)
+// ChrRaces.dbc (8.0.1.27075)
```

Comment:
just cosmetic here. Rename it ChrRaces.db2 :>

Reply:
So be it